homeworks/becseipeter/7_extract_transform/250_market_cap.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import pandas as pd
+
+url = "https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd"
```

Comment:
A vs_currency=usd egy paraméter, ezt egy params dictionary-ből kéne átadni a request-nek, ahogy a headers-szel is csinálod. Onnan ismered fel hogy mi paraméter egy url-ben hogy a "?" után van.